releasenotes/notes/add-repr-str-9a33af0a6ffdca6e.yaml

```diff
@@ -0,0 +1,13 @@
+---
+upgrade:
```

Comment:
Is this an upgrade? or a feature. I thought upgrade were things an end user needed to do to upgrade their installation environment in some way to be compatible with the new release.

Reply:
I see. It then should be `features`. I will update it.